elastic/datadog_checks/elastic/elastic.py

```diff
@@ -26,6 +26,7 @@ class AuthenticationError(requests.exceptions.HTTPError):
 
 class ESCheck(AgentCheck):
     HTTP_CONFIG_REMAPPER = {
+        'aws_service': {'name': 'aws_service', 'default': 'es'},
```

Comment:
Is there any valid use case to change this value?

Reply:
I don't think the user ever would, no, as long as we default to the right one